CONTRIBUTING.md

```diff
@@ -26,6 +26,17 @@ The plugin uses strong types, so be sure to declare `strict_types=1` on new file
 
 For JavaScript we recommend installing ESLint. This plugin includes a [.eslintrc](https://github.com/Parsely/wp-parsely/blob/develop/.eslintrc) file that defines our coding standards.
 
+### Setting up IDE / Code Editor
+
+#### VSCode
+
+This plugin uses the [Multi-root Workspaces](https://code.visualstudio.com/docs/editor/multi-root-workspaces) feature provided by VSCode which setup the editor 
```

Comment:
We have a line break here? Is it needed? Also, we might need some better wording here but we can go ahead for now.

Reply:
Deleted line break.